templates/cluster/openstack-standalone-cp/templates/k0scontrolplane.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+kind: K0sControlPlane
+metadata:
+  name: {{ include "k0scontrolplane.name" . }}
+spec:
+  k0sConfigSpec:
+    args:
+      - --enable-worker
+      - --enable-cloud-provider
+      - --kubelet-extra-args="--cloud-provider=external"
+      - --disable-components=konnectivity-server
+    k0s:
+      apiVersion: k0s.k0sproject.io/v1beta1
+      kind: ClusterConfig
+      metadata:
+        name: k0s
+      spec:
+        api:
+          extraArgs:
+            anonymous-auth: "true"
+        extensions:
+          helm:
+            repositories:
+              - name: openstack
+                url: https://kubernetes.github.io/cloud-provider-openstack/
+            charts:
+              - name: openstack-ccm
+                chartname: openstack/openstack-cloud-controller-manager
+                version: 2.31.1
+                order: 1
+                namespace: kube-system
+                values: |
+                  secret:
+                    enabled: true
+                    name: openstack-cloud-config
+                    create: false
+              - name: openstack-csi
+                chartname: openstack/openstack-cinder-csi
+                version: 2.31.3
```

Comment:
Maybe bump it to the latest version? Same for the CCM.

Reply:
Done.